scripts/release/install.js

```diff
@@ -90,6 +89,7 @@ module.exports = {
 
   redhat: function() {
     console.log("Building rpm package installer")
+    const installerRedhat = require("electron-installer-redhat")
```

Comment:
Perhaps a comment here (and above) as to why these need to required here?

Reply:
I could easily see myself moving this back up top, not knowing why it was here to begin w/.

Reply:
Yep; done